scripts/install/opt/seagate/cortx/motr/common/m0_sns_utils_common.sh

```diff
@@ -46,18 +44,15 @@ M0_NC_REBALANCE=6
 
 CM_OP_REPAIR=1
 CM_OP_REBALANCE=2
-CM_OP_REPAIR_QUIESCE=3
-CM_OP_REBALANCE_QUIESCE=4
-CM_OP_REPAIR_RESUME=5
-CM_OP_REBALANCE_RESUME=6
+# CM_OP_REPAIR_QUIESCE=3
+# CM_OP_REBALANCE_QUIESCE=4
+# CM_OP_REPAIR_RESUME=5
+# CM_OP_REBALANCE_RESUME=6
 CM_OP_REPAIR_STATUS=7
 CM_OP_REBALANCE_STATUS=8
-CM_OP_REPAIR_ABORT=9
-CM_OP_REBALANCE_ABORT=10
+# CM_OP_REPAIR_ABORT=9
+# CM_OP_REBALANCE_ABORT=10
```

Comment:
If we do not need these then please remove the lines.

Reply:
Done.